plugins/connection/zpecloud.py

```diff
@@ -326,23 +326,19 @@ def _wait_job_to_finish(self, job_id: str) -> StringError:
             operation_output_file_url = content.get("output_file", None)
 
             if (
-                operation_status == "Successful"
+                (operation_status == "Successful" or operation_status == "Failed")
                 and operation_output_file_url
                 and len(operation_output_file_url) > 0
             ):
-                self._log_info(f"Job {job_id} finished successfully")
+                self._log_info(f"Job {job_id} finished with status {operation_status}")
                 r = requests.get(operation_output_file_url)
```

Comment:
What if it actually fails and there is no stdout, only stderr?

Reply:
Does it makes sense? What kind of scripts may be executed here?

Reply:
Good point. Stderr is redirected to stdout. In this way, we are going to have something to process.

One case would be if the device fails to upload the output. Then the code will continue quering operation's status up to timeout.